pkg/cloudprovider/plugins.go

```diff
@@ -90,6 +90,11 @@ func InitCloudProvider(name string, configFilePath string) (Interface, error) {
 	var cloud Interface
 	var err error
 
+	if name == "external" {
```

Comment:
I wonder if this distinction is just noise - maybe we should just take "" as "external" ?